CMakeLists.txt

```diff
@@ -1,19 +1,18 @@
 cmake_minimum_required(VERSION 3.10)
 
+project(chip8)
+
 file(GLOB SRC src/*.cc)
 file(GLOB INC inc/*.h)
 
-include(deps/CMakeLists.txt)
-project(chip8)
-
-find_package(sdl2)
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
```

Comment:
What does this include add? Is it a file inside the ${CMAKE_BINARY_DIR}?

Reply:
Yes, include is used to add other files with cmake syntax. Typically *.cmake files.